File: Easy/luckyLong/luckyLong.java
//run as java -ea luckyLong
import java.util.Scanner;

public class luckyLong{
	public static void main(String[] args){
		int T,flag;
		String N;
		Scanner scan = new Scanner(System.in);

		T = scan.nextInt();
		assert(T>=1 && T<=10) : "T must be 1 <= T <= 10";

		while(T>0){
			N = scan.next();
			flag = 0;

			for(int i=0;i<N.length();i++){
				if(N.charAt(i)!='4' && N.charAt(i)!='7'){
					flag++;
				}
			}

			System.out.println(flag);

			T--;
		}
	}
}

File: Easy/luckyLong/luckyLong.c
#include <stdio.h>
#include <assert.h>
#include <string.h>

int main(void){
	int T,flag;
	char N[100003];

	scanf("%d",&T);
	assert(T>=1 && T<=10);

	while(T>0){
		scanf("%s",N);
		flag = 0;

		for(int i=0;i<strlen(N);i++){
			if(N[i]!='4' && N[i]!='7'){
				flag++;
			}
		}

		printf("%d\n",flag);

		T--;
	}
	
	return 0;
}